website/projects/scimaps/src/app/shared/components/profile-gallery/profile-gallery.component.ts

```diff
@@ -17,7 +17,7 @@ export class ProfileGalleryComponent {
   @Input() description = '';
 
   getImageSource(profile: Profile): string {
-    return `assets/${this.directory}/${profile.slug}/image.png`;
+    return `assets/${this.directory}/${profile.slug}/${profile.image}`;
```

Comment:
Resolver should create the url and thus  profile.image is all that is needed for the image src url.

Reply:
Moved this logic to the resolver. Now using a profile.image to directly access the image path.